src/test/resources/TestNG.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="TestNG">
+<suite name="SmokeSuite">
 
-    <test name="Example Test" preserve-order="false">
+    <test name="Smoke Testt" preserve-order="false">
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework18">
+                    <methods>
+                        <include name = 'playSong'/>
+                    </methods>
         </classes>
     </test>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222359966-5c52e2ba-cae3-4f37-a426-aeea0602b958.png)

Hint

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="SmokeSuite">
    <test name="Smoke Testt" preserve-order="false">
        <classes>
            <class name="LoginTests"/>
            <class name="BaseTest"/>
          <class name="Homework18">
            <methods>
              <include name="playSong"/>
            </methods>
          </class>
        </classes>
    </test>
</suite>
```